vendor/github.com/openshift/api/operator/v1/types_kubeapiserver.go

```diff
@@ -29,6 +29,10 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// todo: haggle over the api
+	// +optional
+	RuntimeParameters *RuntimeParameters `json:"runtimeParameters"`
```

Comment:
Instead of this, you could make the value settable only via unsupportedconfigoverrides. It would give you a way to handle the issue in the field if it arises, without growing new, risky, and hard to understand configuration options that will not be needed by most users.